src/tests/unit/tests/DetailsView/components/static-content-details-view.test.tsx

```diff
@@ -57,7 +69,7 @@ class StaticContentDetailsViewPropsBuilder extends BaseDataBuilder<StaticContent
     }
 
     public setupOnToggleClickMock(event: NativeEventStub): StaticContentDetailsViewPropsBuilder {
-        this.onToggleClickMock.setup(click => click(It.isValue(event))).verifiable(Times.once());
+        this.onToggleClickMock.setup(click => click(It.isAny())).verifiable(Times.once());
```

Comment:
We can test with the actual event.
```suggestion
        this.onToggleClickMock.setup(click => click(It.isValue(event))).verifiable(Times.once());
```